CharlieBackend.Panel/wwwroot/js/calendar.js

```diff
@@ -0,0 +1,36 @@
+﻿var seGroup = null;
+var seMentor = null;
+var seTheme = null;
+var seTime = null;
+var seEventOccurrenceId = null;
+var seSingleEventId = null;
```

Comment:
creating var variable outside of functions makes its global scope visible, I don't think that it was ur purpose